lib/perl/Genome/Model/Tools/SpeedSeq/Realign.pm

```diff
@@ -0,0 +1,67 @@
+package Genome::Model::Tools::SpeedSeq::Realign;
+
+use strict;
+use warnings;
+
+use Genome;
+
+class Genome::Model::Tools::SpeedSeq::Realign {
+    is => 'Genome::Model::Tools::SpeedSeq::AlignBase',
+    has_input => [
+        bams => {
+            is => 'Text',
+            doc => 'BAM file(s) (must contain read group tags)',
+            is_many => 1,
+            example_values => ['in.bam'],
+        },
+    ],
+    has_param => [
+        rename_reads => {
+            is => 'Boolean',
+            doc => 'rename reads for smaller file size',
+            is_optional => 1,
+        },
+    ],
+};
+
+sub execute {
+    my $self = shift;
+
+    # OPTIONS aka params
+    my $options = $self->_resolve_options_string();
+    if ($self->rename_reads) {
+        $options .= ' -n';
+    }
+
+    # INPUTS
+    my @input_files;
+    my $inputs_string = $self->reference_fasta;
+    push @input_files, $self->reference_fasta;
+    for my $bam ($self->bams) {
+        push @input_files, $bam;
+        $inputs_string .= ' '. $bam;
+    }
+    if ($self->config_file) {
+        push @input_files, $self->config_file;
+    }
+
+    # COMMAND
+    my $cmd = $self->speedseq_path .' realign '. $options .' '. $inputs_string;
+
+    # OUTPUTS
+    my $basename = 'in.realign';
```

Comment:
It's a bit strange how the align and realign commands resolve the output_prefix.  'align' will use the first FASTQ input as the prefix by default.  The 'realign' command will use the string 'in.realign'.  I'm not sure if the GMT version should have any default value.  Can I override the default_value in each subcommand?  Feedback is much appreciated.